common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -645,9 +645,14 @@
         public static final String BROKER_ACTIVITY_NAME = "broker.activity.name";
 
         /**
-         * The maximum broker protocol version that common supports.
+         * The newest Msal-To-Broker protocol version.
          */
-        public static final String BROKER_PROTOCOL_VERSION_CODE = "5.0";
+        public static final String MSAL_TO_BROKER_PROTOCOL_VERSION_CODE = "5.0";
+
+        /**
+         * The newest BrokerAPI-To-Broker protocol version.
+         */
+        public static final String BROKER_API_TO_BROKER_PROTOCOL_VERSION_CODE = "1.0";
```

Comment:
Can we describe this via a schema rather than a number?

Reply:
We definitely can, but what would be the benefit over using a number?

Reply:
The bundle itself is technically a kind of schema. I think we can add metadata to the bundle directly.